src/conversions.jl

```diff
@@ -7,8 +7,16 @@ using ..LeapSeconds
 
 
 """
+<<<<<<< HEAD
```

Comment:
There is still this merge conflict to fix.

I think you're discovering why it's a very bad idea to work on `master` branch :smile:

Reply:
Yes! I'm regretting it now! I'll fix this asap! And then, after getting a merge I'll clone the master branch again and then start working with new functions(not master this time at all), sounds good?

Reply:
You should use `master` only to keep in sync with upstream `master` and start a new feature branch from the most recent `master`.  That's how `master` is usually used on GitHub.

Regarding how to fix your issue, I think that a hard reset on `master` using upstream as `origin` should be enough (`git reset --hard origin/master`, if `origin` remote is this repository).  However, if you feel more safe, just clone again this repository.  In any case, do this *after* the PR will be merged :wink:

Reply:
I tried to fix it and it got worse! :disappointed: